lm_eval/tasks/race/race.yaml

```diff
@@ -5,9 +5,9 @@ dataset_path: EleutherAI/race
 dataset_name: high
 output_type: multiple_choice
 test_split: test
-create_choices: !function preprocess_race.create_choices
 doc_to_text: !function preprocess_race.doc_to_text
 doc_to_target: !function preprocess_race.doc_to_target
+doc_to_choice: !function preprocess_race.doc_to_choice
```

Comment:
I don't see `doc_to_choice` in preprocess_race.py I think

Reply:
Ohh, might have not added that. Will fix.